domain_source/domain_source.module

```diff
@@ -8,11 +8,6 @@
 use Drupal\Core\Entity\EntityInterface;
 
 /**
- * Defines the name of the source domain field.
- */
-define('DOMAIN_SOURCE_FIELD', 'field_domain_source');
```

Comment:
As you can tell, I'm having trouble getting my fork to compare to your main line.